internal/commands/test.go

```diff
@@ -159,6 +161,7 @@ func NewTestCommand(ctx context.Context) *cobra.Command {
 	cmd.Flags().Bool("no-color", false, "Disable color when printing")
 	cmd.Flags().Bool("suppress-exceptions", false, "Do not include exceptions in output")
 	cmd.Flags().Bool("all-namespaces", false, "Test policies found in all namespaces")
+	cmd.Flags().Bool("quiet", false, "Disable successful test output")
```

Comment:
this isn't actually the same behavior as it's provided in verify,
e.g
```
./conftest verify -p examples/kubernetes
6 tests, 6 passed, 0 warnings, 0 failures, 0 exceptions, 0 skipped
./conftest verify -p examples/kubernetes --quiet 
```

vs
```
./conftest test -p examples/kubernetes/policy/deny.rego examples/kubernetes/deployment.yaml
2 tests, 2 passed, 0 warnings, 0 failures, 0 exceptions
./conftest test -p examples/kubernetes/policy/deny.rego examples/kubernetes/deployment.yaml --quiet
2 tests, 2 passed, 0 warnings, 0 failures, 0 exceptions
```

hence I'd recommend renaming it as something similar to `suppress-no-policies-found` to be more specific or refactoring it to extend hiding all the outputs when the exit code is 0 (success)

Reply:
You are right, should be fixed now. Added the suppress-no-policies-found option as well as that is the one I'm really interested in :)